src/main/java/com/olgaboiar/mint/handlers/FileHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.olgaboiar.mint.handlers;
+
+import com.olgaboiar.mint.Request;
+import com.olgaboiar.mint.Response;
+import com.olgaboiar.mint.ResponseGenerator;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static com.olgaboiar.mint.Constants.STATUS_CODE_200;
+
+public class FileHandler implements IHandler {
+    @Override
+    public Response handleRequest(Request request) throws IOException {
+        String file = "." + request.getRequestedFile();
+        List<String> fileContent = Files.readAllLines(Paths.get(file));
+        String body = String.join("", fileContent);
+        if (request.getMethod().equals("HEAD")) {
```

Comment:
I might just break this out into a separate handler.

Reply:
refactored in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/90a3ba79ceba201a9c960f5d37f84d886370a826